gradle/libs.versions.toml

```diff
@@ -55,6 +55,7 @@ gms-play-services-auth = "com.google.android.gms:play-services-auth:20.4.1"
 lifecycle-livedataKtx = { module = "androidx.lifecycle:lifecycle-livedata-ktx", version.ref = "lifecycle" }
 lifecycle-runtimeKtx = { module = "androidx.lifecycle:lifecycle-runtime-ktx", version.ref = "lifecycle" }
 lifecycle-viewmodel-compose = { module = "androidx.lifecycle:lifecycle-viewmodel-compose", version.ref = "lifecycle" }
+lifecycle-runtime-compose = "androidx.lifecycle:lifecycle-runtime-compose:2.6.0"
```

Comment:
I think is already introduced by another PR merged yesterday - [Reference](https://github.com/droidconKE/droidconKE2023Android/blob/a0a47f21b1ec93684ed49fbc0a8aed27e8877d38/gradle/libs.versions.toml#L46)... You can update your branch to get the lib so that we avoid duplicates.

Reply:
he can `git rebase main` remove duplicate and push update to his remote branch

Reply:
I've removed the duplicate dependencies 
Check the latest commit